server/src/main/java/com/defold/extender/WebSecurityConfig.java

```diff
@@ -69,7 +58,11 @@ protected void configure(HttpSecurity http) throws Exception {
 
 	@Override
 	protected void configure(AuthenticationManagerBuilder auth) throws Exception {
-		Properties users = loadUsers();
-		auth.userDetailsService(new InMemoryUserDetailsManager(users));
+		auth.userDetailsService(userDetailsManager);
+	}
+
+	@Bean
```

Comment:
Expose the user details manager as a Spring Bean so that we can access it from the UserUpdateService